src/main/java/flaxbeard/steamcraft/block/BlockSteamCharger.java

```diff
@@ -22,8 +17,16 @@
 import net.minecraft.world.IBlockAccess;
 import net.minecraft.world.World;
 import net.minecraftforge.common.util.ForgeDirection;
-
-import java.util.Random;
+import tconstruct.library.tools.ToolCore;
```

Comment:
Already tested a build to make sure, it doesn't cause an issue, at least in that build.